common/errors_test.go

```diff
@@ -82,6 +83,7 @@ func TestDownstreamError_CreateDownstreamError_UnexpectedResponse(t *testing.T)
 	r.WriteHeader(http.StatusInternalServerError)
 
 	resp := r.Result()
+	defer resp.Body.Close()
```

Comment:
This change has been fixed as part of separate pr - https://github.com/anz-bank/sysl-go/pull/253